P3826/P3826.md

```diff
@@ -1348,15 +1347,15 @@ follows:]{.ednote}
 >     friend bool operator==(const task_scheduler& lhs, const Sch& rhs) noexcept;
 >
 >   private:
->     shared_ptr<@[`void`]{.rm}[`parallel_scheduler_backend`]{.add}@> sch_; // exposition only
+>     shared_ptr<@[`void`]{.rm}[`parallel_scheduler_backend`]{.add}@> @_`sch_`_@; // exposition only
 >                                                      @[_`// see [exec.sysctxrepl.psb]`_]{.add}@
 >   };
 > }
 > ```
 >
 > 1. `task_scheduler` is a class that models `scheduler` ([exec.sched]{.sref}). Given an object
 >    `s` of type `task_scheduler`, let ```@_`SCHED`_@(s)```{.cpp} be [the `sched_` member
```

Comment:
I wonder if we should also make `sched_` italic and say that is not just a member but exposition only member and then also make `sched_` italic all over the places and add `// exposition only` comment to the line 1382

Reply:
yes, i'll make that change, thanks